app/api/v2/views/AdminViews.py

```diff
@@ -0,0 +1,91 @@
+from ..models.AdminModels import UserOrders
+from ..models.UserModels import Order
+from flask_restful import Resource
+from flask import make_response, jsonify, request, abort
+
+
+class Admin_all_Orders(Resource):
```

Comment:
Use a decorator to check whether a user is an admin or not. You should not hardcode the roles in the resource classes